backend/models/bill.js

```diff
@@ -17,4 +22,19 @@ const billSchema = new mongoose.Schema({
 
 const Bill = mongoose.model('Bill', billSchema);
 
+const newBill = new Bill({
```

Comment:
why are we instantiating a bill object here? this looks like code that belongs in the api. please explain to me what `newBill` is for.

Reply:
The new bill object was for testing and saving a bill to the database. I shouldn't have left it in the bill.js.